docs/apis-tools/operate-api/overview.md

```diff
@@ -42,6 +42,13 @@ For SaaS: `https://${REGION}.operate.camunda.io/${CLUSTER_ID}/swagger-ui.html`,
 Find your **region Id** and **cluster Id** under **Connection information** in your client credentials (revealed when you click on your client under the **API** tab within your cluster).
 :::
 
+#### Limitation for Operate API v.1
+
+The Swagger UI may return incorrect `id` and `key` values for data created on Zeebe partitions >= 4. In case precise ids are going to be copied and used in another queries, consider using another REST API client, e.g. Postman or `curl` command.
+The reason is that Zeebe may create `key` values that are bigger than JavaScript's `Number.MAX_SAFE_INTEGER` (2^53-1) and Swagger UI may display such values incorrectly.
```

Comment:
```suggestion
Swagger UI may display incorrect `id` and `key` values for data created on Zeebe partitions >= 4. In case precise ids are going to be copied and used in other queries, consider using another REST API client, e.g. Postman or the `curl` command.
The reason is that Zeebe may create `key` values that are greater than JavaScript's `Number.MAX_SAFE_INTEGER` (2^53-1) and Swagger UI may display such values incorrectly.
```